pkg/kubelet/cm/devicemanager/pod_devices.go

```diff
@@ -244,14 +249,16 @@ func (pdev *podDevices) deviceRunContainerOptions(podUID, contName string) *Devi
 	mountsMap := make(map[string]string)
 	envsMap := make(map[string]string)
 	annotationsMap := make(map[string]string)
+	cdiDevicesMap := make(map[string]struct{})
```

Comment:
using a `struct{}` as value in a map suggest this is actually a set. Is that the case?

Reply:
This could be condidered a set, I suppose. Note that the only intent of this variable is to ensure that we de-duplicate CDI devices. Since these are not key-value pairs, there are no other values that could possibly be overwritten. Let me see if switching to a set cleans things up at all.

Reply:
Yup. My comment is nonblocking for alpha anyway, we still can improve later.

Reply:
A string set would probably be nicer here.

Reply:
I have used a `Set[string]` here instead. I don't think it cleans things up too much, but checking `.Has` is at least a little easier to reason about than the map semantics.